File: Search_Value.java
public int searchValue(Node head, int value) {
    Node current = head;
    int position = 0;

    while (current != null) {
        if (current.data == value) {
            return position;
        }
        current = current.link;
        position++;
    }

    return -1; // Value not found
}

File: Search_Value.cpp
int searchValue(Node* head, int value) {
    Node* current = head;
    int position = 0;

    while (current != nullptr) {
        if (current->data == value) {
            return position;
        }
        current = current->link;
        position++;
    }

    return -1; // Value not found
}
